components/BottomSheet/tests/snapshot/MDCBottomSheetSnapshotTests.m

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialBottomSheet.h"
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+#import "MaterialSnapshot.h"
+
+/** Snapshot test for @c MDCBottomSheetController. */
+@interface MDCBottomSheetSnapshotTests : MDCSnapshotTestCase
```

Comment:
Nit: Please name this `MDCBottomSheetControllerSnapshotTests` so it's clear it's a test for the controller.

Reply:
Done